### ISSUE TITLE
Fix snappy installation on Mac

### ISSUE THREAD

**Issue by sharattadimalla** [capitalone/DataProfiler#970]
**Please provide the issue you face regarding the documentation**

### Problem Statement

Current documentation step does not yield successful setup of snappy. 
Section referenced in documentation - https://capitalone.github.io/DataProfiler/docs/0.10.1/html/install.html#snappy-installation

### Proposed Solution

Update documentation as per the official procedure on `python-snappy`. 
Reference - https://github.com/andrix/python-snappy#frequently-asked-questions

```
brew install snappy && CPPFLAGS="-I/usr/local/include -L/usr/local/lib" pip install python-snappy
```
I am looking to contribute as a way to get started.

**Comment by taylorfturner** [comment]
Hey @sharattadimalla! Thanks for opening the issue and raising this -- we welcome pull requests, since you mentioned you are looking for ways to contribute. 

Check out our contribution guide [here](https://github.com/capitalone/DataProfiler/blob/main/.github/CONTRIBUTING.md#creating-pull-requests) for creating pull requests.

**Comment by taylorfturner** [comment]
@sharattadimalla just checking on the issue. Any questions about contributing or the like? Thanks for the interest!

**Comment by jacob-buehler** [comment]
fixed and merged #1010
